sdk/adp/Azure.AI.AutonomousDevelopmentPlatform/src/properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This folder should be capital `Properties`

Reply:
done